### PR TITLE
Invert the way RA locations are configurable

### PR DESCRIPTION
**Pull request by arothuis**
Instead of configuring which institutions should show personal RA details, it can now be configured which institutions want to specify RA locations to show.

Naming has been adjusted to this practice.

### CODE REVIEW

File: app/DoctrineMigrations/Version20160622160146.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE TABLE institution_with_ra_locations (institution VARCHAR(255) NOT NULL, PRIMARY KEY(institution)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
+        $this->addSql('DROP TABLE institution_with_ra_locations');
```
Comment (rjkip) [inline app/DoctrineMigrations/Version20160622160146.php]: Eh?

File: src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml [hunk 2]
```diff
@@ -110,9 +110,9 @@ institution_listing:
     methods:   [GET]
     condition: "request.headers.get('Accept') matches '/^application\\\\/json($|[;,])/'"
 
-institution_has_personal_ra_locations:
-    path: /has-personal-ra-locations/{institution}
-    defaults: { _controller: SurfnetStepupMiddlewareApiBundle:InstitutionWithPersonalRaDetails:hasPersonalRaDetails }
+institution_shows_ra_locations:
+    path: /shows-personal-ra-locations/{institution}
```
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml]: Why "shows"? The institution doesn't show RA locations. Sounds like an RA GUI detail.

Also: `personal` is still in the path.

File: app/DoctrineMigrations/Version20160622160146.php (hunk 1 above)
Comment (arothuis) [inline app/DoctrineMigrations/Version20160622160146.php]: LOL string replace failure.

File: src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml (hunk 2 above)
Comment (arothuis) [inline src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml]: But showing (or not) *is* something that is specifically configured in MW.

Also: string replace failure :(

File: app/DoctrineMigrations/Version20160622160146.php (hunk 1 above)
Comment (rjkip) [inline app/DoctrineMigrations/Version20160622160146.php]: Pls test you're work m8

Comment (arothuis) [inline app/DoctrineMigrations/Version20160622160146.php]: I replaced, I tested, I merged develop, needed to replace again (I should have rebased on develop) and now I cry.

Also: it's "your".

**Comment by arothuis** [comment]
The logic of the VettingLocationService should be inverted as well.

File: src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml (hunk 2 above)
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml]: Configuration option is `institutions_with_ra_locations`. No "institutions showing RA locations" in there. ;) So I disagree.

File: src/Surfnet/StepupMiddleware/ApiBundle/Service/VettingLocationService.php [hunk 3]
```diff
@@ -32,7 +32,7 @@
 final class VettingLocationService implements VettingLocationServiceInterface
 {
     /**
-     * @var InstitutionWithPersonalRaDetailsService
+     * @var InstitutionWithRaLocationsService
      */
     private $personalRaDetailsService;
```
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/ApiBundle/Service/VettingLocationService.php]: Rename

**Pull request merged**